pkg/connector/connector.go

```diff
@@ -67,16 +67,19 @@ func New(ctx context.Context, username string, password string, instanceURL stri
 		return nil, err
 	}
 
-	token, err := jamf.CreateBearerToken(ctx, username, password, instanceURL)
+	client := jamf.NewClient(
+		uhttp.NewBaseHttpClient(httpClient),
+		"",
+		instanceURL,
+	)
+
+	token, err := client.CreateBearerToken(ctx, username, password)
 	if err != nil {
 		return nil, fmt.Errorf("jamf-connector: failed to get token: %w", err)
 	}
+	client.SetBearerToken(token)
 
-	baseUrl := fmt.Sprintf("%s/JSSResource", instanceURL)
```

Comment:
Where do we append "/JSSResource" to the instance URL?

Reply:
In `client.go`, I explicitly include `/JSSResource/` in every relevant path.
```go
const (
	accountUrlPath    = "/JSSResource/accounts/userid/%d"
	accountsUrlPath   = "/JSSResource/accounts"
	authUrlPath       = "/api/v1/auth"
	groupUrlPath      = "/JSSResource/accounts/groupid/%d"
	sitesUrlPath      = "/JSSResource/sites"
	tokenUrlPath      = "/api/v1/auth/token" //nolint:golint,gosec // not a token
	userGroupUrlPath  = "/JSSResource/usergroups/id/%d"
	userGroupsUrlPath = "/JSSResource/usergroups"
	userUrlPath       = "/JSSResource/users/id/%d"
	usersUrlPath      = "/JSSResource/users"
)
```